easybuild/easyconfigs/l/LibUUID/LibUUID-1.0.3-GCCcore-6.4.0.eb

```diff
@@ -11,12 +11,18 @@ toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
 
 source_urls = [SOURCEFORGE_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['46af3275291091009ad7f1b899de3d0cea0252737550e7919d17237997db5644']
+patches = [('libuuid.sym', '%(builddir)s')]
```

Comment:
@ocaisa You'll need to include this in the PR.

But maybe it's better to download this on the fly, by listing it in `sources` instead?

Reply:
Can you give me the correct syntax, it's a hunt to figure out how to use `extract_cmd`. Is it
```
source_urls = [
    SOURCEFORGE_SOURCE,
    'https://raw.githubusercontent.com/karelzak/util-linux/master/libuuid/src',
]
sources = [
    SOURCELOWER_TAR_GZ,
    {'filename': 'libuuid.sym', 'extract_cmd': 'mv libuuid.sym %(builddir)s')]
checksums = [
    '46af3275291091009ad7f1b899de3d0cea0252737550e7919d17237997db5644',  # libuuid-1.0.3.tar.gz
    '5932866797560bf5822c3340b64dff514afb6cef23a824f96991f8e9e9d29028',  # libuuid.sym
]
```

Reply:
`cp %s %(builddir)s` as extract command should do it

the `%s` gets replaced with the full path download location of `libuuid.sym`